validator/accounts/account.go

```diff
@@ -149,20 +149,19 @@ func CreateValidatorAccount(path string, passphrase string) (string, string, err
 
 	if path == "" {
 		path = DefaultValidatorDir()
-	}
-	log.Infof("Keystore path to save your private keys (default: %q):", path)
-	reader := bufio.NewReader(os.Stdin)
-	text, err := reader.ReadString('\n')
-	if err != nil {
-		log.Fatal(err)
-		return path, passphrase, err
-	}
-	if text = strings.Replace(text, "\n", "", -1); text != "" {
-		path = text
-	}
-
-	if err := NewValidatorAccount(path, passphrase); err != nil {
-		return "", "", errors.Wrapf(err, "could not initialize validator account")
+		log.Infof("Please specify a keystore path to save your private keys (default: %q):", path)
+		reader := bufio.NewReader(os.Stdin)
+		text, err := reader.ReadString('\n')
+		if err != nil {
+			log.Fatal(err)
+			return path, passphrase, err
+		}
+		if text = strings.Replace(text, "\n", "", -1); text != "" {
+			path = text
+		}
+		if err := NewValidatorAccount(path, passphrase); err != nil {
```

Comment:
Thanks for doing this! I think this part should be outside of the `if` no? Or else it won't create the keys unless a default is set.